src/main.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+class Todo  {
+    constructor() {
+        this.todoPoints = [];
+    }
+
+    addItem(title, priority) {
+        let todoPoint = {};
+        todoPoint.title = title;
+        todoPoint.priority = priority;
+        todoPoint.id = +Math.random().toString().substr(2, 4);
```

Comment:
An interesting option))

but if you use random numbers, you need to check for repeat id with id we had before